docs/dex-limit-order-post.md

```diff
@@ -0,0 +1,105 @@
+# Decentralized Exchange on Ergo
+
+Ergo have expressive smart contracts and transaction model which allows the implementation of trustless DEX protocol, in which signed buy and sell orders can be put into the blockchain independently by buyers and sellers. An off-chain matching service can observe the Ergo blockchain, find matching orders, and submit the swap transaction without knowing any secrets. The matching can be incentivized by DEX reward paid as part of a swap transaction. Anyone who first discover the match of the two orders can create the swap transaction and get a reward in ERGs. Partial matching is supported, meaning that target (buy/sell) order can be executed partially, in which case a new "residual" order(box) has to be created in the same swap transaction. Any order can be canceled anytime by the "owner."
```

Comment:
transaction model => transactional model

Reply:
allows the implementation => allows for an implementation

Reply:
Thanks! Fixed.